.github/workflows/slow.yml

```diff
@@ -42,3 +43,10 @@ jobs:
       - name: Test with pytest
         run: |
           py.test -m slow
+          slack_api:
+      - name: Slack notify via api
+        uses: up9cloud/action-notify@master
```

Comment:
When I first read this, I was concerned that we were using a github action that only had 1 star on Github and wondered why we weren't using the official [slack-send](https://github.com/marketplace/actions/slack-send) action.

After reading into it a bit, I realized that `slack-send` is lower level, more flexible, but requires more setup - we'd have to craft the messages ourselves from scratch.

I haven't seen what `up9cloud/action-notify` messages look like, but I'm hoping they're nicely formatted out-of-the-box.

This is just a comment, no change necessary.

Reply:
No big deal on how it looks, it just links to the action and says: success or failure. Check the slack channel for an example, since it has already ran once. I can also change the app image (which I created in Slack) to look prettier but I have no idea in particular as to what it should be :))